bin/psrlint

```diff
@@ -4,11 +4,10 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-//echo 'what is your name?: ';
-//$name = trim(fgets(STDIN));
-//
-//$user = new PhpPackage\User($name);
-//
-//var_dump($user);
-
-print_r($argv[]);
\ No newline at end of file
+use function \HexletPsrLinter\getFunctions;
+
+$filename = $argv[1];
```

Comment:
Библиотеку добавил.

On Fri, Aug 5, 2016 at 10:02 AM, Roman Lakhtadyr <notifications@github.com>
wrote:

> In bin/psrlint
> <https://github.com/maximtop/hexlet-psr-linter/pull/1#discussion_r73650045>
> :
>
> > \ No newline at end of file
> > +use function \HexletPsrLinter\getFunctions;
> > +
> > +$filename = $argv[1];
>
> Работать напрямую с суперглобальными массивами - очень нехорошая практика.
> Для обработки аргументов существуют специальные в библиотеки, например,
> nategood/commando.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/maximtop/hexlet-psr-linter/pull/1/files/de77fd2c026aec74dad587f6eb61c5e151b5f7c4#r73650045>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQePwfOCgRdceHYX7luQXMwkunH2Bnoaks5qcuADgaJpZM4JdWB2>
> .
>



-- 
Sincerely yours,
Maxim Nikolaevich
mob. 8-926-934-08-25
email: maximtop@gmail.com